nntrainer/layers/embedding.cpp

```diff
@@ -104,17 +104,19 @@ void EmbeddingLayer::forwarding(bool training) {
     float *in_data = input_.getAddress(b * input_.getDim().getFeatureLen());
 
     for (unsigned int i = 0; i < in_length; ++i) {
-      if (in_data[i] < 0)
+      if (in_data[i] > in_dim) {
```

Comment:
if (`in_data[i] == in_dim`) is allowed, I wonder if `in_data[i] == 0` is defined in this case? for keras, only positive integer seems allowed.